patches/0002-Conns-user-agent.patch

```diff
@@ -1,34 +0,0 @@
-From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
-From: Daniel Bradley <daniel@pulumi.com>
-Date: Fri, 4 Nov 2022 17:03:31 +0000
-Subject: [PATCH 02/48] Conns user agent
-
-Replace the useragent used for AWS client connections with a
-Pulumi-flavoured one.
-
-diff --git a/internal/conns/conns.go b/internal/conns/conns.go
-index 954266ba7a..404a654792 100644
---- a/internal/conns/conns.go
-+++ b/internal/conns/conns.go
-@@ -12,7 +12,6 @@ import (
- 	awsbase "github.com/hashicorp/aws-sdk-go-base/v2"
- 	awsbasev1 "github.com/hashicorp/aws-sdk-go-base/v2/awsv1shim/v2"
- 	"github.com/hashicorp/terraform-provider-aws/internal/types"
--	"github.com/hashicorp/terraform-provider-aws/version"
- )
- 
- // ServicePackage is the minimal interface exported from each AWS service package.
-@@ -80,10 +79,10 @@ func NewSessionForRegion(cfg *aws_sdkv1.Config, region, terraformVersion string)
- 
- func StdUserAgentProducts(terraformVersion string) *awsbase.APNInfo {
```

Comment:
This is not needed anymore? Looks like this was fixing up User Agent somewhere in one of the clients?

Reply:
ahh good catch! It looks like this configuration was moved somewhere else. I'll create another patch.

Reply:
ok I added another patch https://github.com/pulumi/pulumi-aws/pull/3792/files#diff-fec171c8e8ff659477dbbd56f2c37c3c617a42bcd723bdc592099abfa111b2c5

Reply:
Yes it looks like an untested patch .. maybe one day we'll backfill the test for it so it'd remind us.